CHANGELOG.md

```diff
@@ -14,6 +14,12 @@ Types of changes are:
 
 ## [Unreleased]
 
+## [0.16.1] - 2022-09-01
```

Comment:
Please update the links at the end of this file as well :pray:

Reply:
[9d2bac4](https://github.com/radeklat/delfino/pull/23/commits/9d2bac4a5a307017e5c58ba56ab7b24927866f5a)

done!